src/cmd/troubleshoot/builder.go

```diff
@@ -87,135 +82,160 @@ func (builder CommandBuilder) buildRun() func(*cobra.Command, []string) error {
 			return err
 		}
 
-		k8scluster, err := builder.GetCluster(kubeConfig)
-		if err != nil {
-			return err
-		}
-
-		apiReader := k8scluster.GetAPIReader()
-
 		log := NewTroubleshootLoggerToWriter(os.Stdout)
 
-		RunTroubleshootCmd(context.Background(), log, apiReader, namespaceFlagValue, *kubeConfig)
+		RunTroubleshootCmd(cmd.Context(), log, namespaceFlagValue, kubeConfig)
 		return nil
 	}
 }
-
-func RunTroubleshootCmd(ctx context.Context, log logr.Logger, apiReader client.Reader, namespace string, kubeConfig rest.Config) {
-	troubleshootCtx := troubleshootContext{
-		context:       ctx,
-		apiReader:     apiReader,
-		httpClient:    &http.Client{},
-		namespaceName: namespace,
-		kubeConfig:    kubeConfig,
-		baseLog:       log,
+func RunTroubleshootCmd(ctx context.Context, log logr.Logger, namespaceName string, kubeConfig *rest.Config) {
```

Comment:
could you please insert  empty line before `func RunTroubleshootCmd`

Reply:
done